{
  "cancel_ci": false,
  "cancel_ci_branch": false,
  "close": false,
  "comments": [
    "The test `ansible-test sanity --test validate-modules` [[explain](https://docs.ansible.com/ansible/devel/dev_guide/testing/sanity/validate-modules.html)] failed with 1 error:\n```\nplugins/modules/cloud/misc/proxmox_interfaces.py:0:0: return-syntax-error: RETURN.msg.elements: not a valid value for dictionary value @ data['msg']['elements']. Got 'string'\n```\n<!-- job_id: 23303 -->\n<!-- job_id: 5d06d1d8278c42272b957b0fb92d6fa9 -->\n<!-- job_id: None -->\nThe test `ansible-test sanity --test import --python 2.6` [[explain](https://docs.ansible.com/ansible/devel/dev_guide/testing/sanity/import.html)] failed with 1 error:\n```\nplugins/modules/cloud/misc/proxmox_interfaces.py:337:36: traceback: SyntaxError: invalid syntax\n```\n<!-- job_id: 23303 -->\n<!-- job_id: 38ec78ca0caeba4e841794205460d655 -->\n<!-- job_id: None -->\nThe test `ansible-test sanity --test compile --python 2.6` [[explain](https://docs.ansible.com/ansible/devel/dev_guide/testing/sanity/compile.html)] failed with 1 error:\n```\nplugins/modules/cloud/misc/proxmox_interfaces.py:337:36: SyntaxError: present_nics = {nic['iface'] for nic in nics}\n```\n<!-- job_id: 23303 -->\n<!-- job_id: 6da3d82ccead7a5020f7763183374a54 -->\n<!-- job_id: None -->\nThe test `ansible-test sanity --test import --python 2.6` [[explain](https://docs.ansible.com/ansible/devel/dev_guide/testing/sanity/import.html)] failed with 1 error:\n```\nplugins/modules/cloud/misc/proxmox_interfaces.py:337:36: traceback: SyntaxError: invalid syntax\n```\n<!-- job_id: 23303 -->\n<!-- job_id: 38ec78ca0caeba4e841794205460d655 -->\n<!-- job_id: None -->\nThe test `ansible-test sanity --test compile --python 2.6` [[explain](https://docs.ansible.com/ansible/devel/dev_guide/testing/sanity/compile.html)] failed with 1 error:\n```\nplugins/modules/cloud/misc/proxmox_interfaces.py:337:36: SyntaxError: present_nics = {nic['iface'] for nic in nics}\n```\n<!-- job_id: 23303 -->\n<!-- job_id: 6da3d82ccead7a5020f7763183374a54 -->\n<!-- job_id: None -->\nThe test `ansible-test sanity --test import --python 2.6` [[explain](https://docs.ansible.com/ansible/devel/dev_guide/testing/sanity/import.html)] failed with 1 error:\n```\nplugins/modules/cloud/misc/proxmox_interfaces.py:337:36: traceback: SyntaxError: invalid syntax\n```\n<!-- job_id: 23303 -->\n<!-- job_id: 38ec78ca0caeba4e841794205460d655 -->\n<!-- job_id: None -->\nThe test `ansible-test sanity --test compile --python 2.6` [[explain](https://docs.ansible.com/ansible/devel/dev_guide/testing/sanity/compile.html)] failed with 1 error:\n```\nplugins/modules/cloud/misc/proxmox_interfaces.py:337:36: SyntaxError: present_nics = {nic['iface'] for nic in nics}\n```\n<!-- job_id: 23303 -->\n<!-- job_id: 6da3d82ccead7a5020f7763183374a54 -->\n<!-- job_id: None -->\nThe test `ansible-test sanity --test validate-modules` [[explain](https://docs.ansible.com/ansible/devel/dev_guide/testing/sanity/validate-modules.html)] failed with 1 error:\n```\nplugins/modules/cloud/misc/proxmox_interfaces.py:0:0: return-syntax-error: RETURN.msg.elements: not a valid value for dictionary value @ data['msg']['elements']. Got 'string'\n```\n<!-- job_id: 23303 -->\n<!-- job_id: 5d06d1d8278c42272b957b0fb92d6fa9 -->\n<!-- job_id: None -->\nThe test `ansible-test sanity --test validate-modules` [[explain](https://docs.ansible.com/ansible/devel/dev_guide/testing/sanity/validate-modules.html)] failed with 1 error:\n```\nplugins/modules/cloud/misc/proxmox_interfaces.py:0:0: return-syntax-error: RETURN.msg.elements: not a valid value for dictionary value @ data['msg']['elements']. Got 'string'\n```\n<!-- job_id: 23303 -->\n<!-- job_id: 5d06d1d8278c42272b957b0fb92d6fa9 -->\n<!-- job_id: None -->\nThe test `ansible-test sanity --test import --python 2.6` [[explain](https://docs.ansible.com/ansible/devel/dev_guide/testing/sanity/import.html)] failed with 1 error:\n```\nplugins/modules/cloud/misc/proxmox_interfaces.py:337:36: traceback: SyntaxError: invalid syntax\n```\n<!-- job_id: 23303 -->\n<!-- job_id: 38ec78ca0caeba4e841794205460d655 -->\n<!-- job_id: None -->\nThe test `ansible-test sanity --test compile --python 2.6` [[explain](https://docs.ansible.com/ansible/devel/dev_guide/testing/sanity/compile.html)] failed with 1 error:\n```\nplugins/modules/cloud/misc/proxmox_interfaces.py:337:36: SyntaxError: present_nics = {nic['iface'] for nic in nics}\n```\n<!-- job_id: 23303 -->\n<!-- job_id: 6da3d82ccead7a5020f7763183374a54 -->\n<!-- job_id: None -->\nThe test `ansible-test sanity --test validate-modules` [[explain](https://docs.ansible.com/ansible/devel/dev_guide/testing/sanity/validate-modules.html)] failed with 1 error:\n```\nplugins/modules/cloud/misc/proxmox_interfaces.py:0:0: return-syntax-error: RETURN.msg.elements: not a valid value for dictionary value @ data['msg']['elements']. Got 'string'\n```\n<!-- job_id: 23303 -->\n<!-- job_id: 5d06d1d8278c42272b957b0fb92d6fa9 -->\n<!-- job_id: None -->\n\n[click here for bot help](https://github.com/ansible/ansibullbot/blob/devel/ISSUE_HELP.md)\n<!--- boilerplate: shippable_test_result --->"
  ],
  "merge": false,
  "newlabel": [
    "needs_revision"
  ],
  "open": false,
  "rebuild": false,
  "rebuild_failed": false,
  "uncomment": [],
  "unlabel": [
    "needs_ci",
    "new_contributor",
    "new_module"
  ]
}